docs/content/installation/csi_ug_requirements.md

```diff
@@ -37,33 +37,60 @@ Perform these steps for each worker node in Kubernetes cluster to prepare your e
+   Apply the YAML file.
 
    `oc apply -f 99-ibm-attach.yaml`
+
+3. Configure storage system connectivity.
+
+    1.  Define the host of each Kubernetes node on the relevant storage systems with the valid WWPN (for Fibre Channel) or IQN (for iSCSI) of the node.
+
+    2.  For Fibre Channel, configure the relevant zoning from the storage to the host.
     
-3. If needed, enable support for volume snapshots (FlashCopy® function) on your Kubernetes cluster.
+4. (Optional) If planning on using volume snapshots (FlashCopy® function), enable support on your Kubernetes cluster.
 
    For more information and instructions, see the Kubernetes blog post, [Kubernetes 1.20: Kubernetes Volume Snapshot Moves to GA](https://kubernetes.io/blog/2020/12/10/kubernetes-1.20-volume-snapshot-moves-to-ga/).
 
    Install both the Snapshot CRDs and the Common Snapshot Controller once per cluster.
 
-   The instructions and relevant yaml files to enable volume snapshots can be found at: [https://github.com/kubernetes-csi/external-snapshotter#usage](https://github.com/kubernetes-csi/external-snapshotter#usage)
+   The instructions and relevant YAML files to enable volume snapshots can be found at: [https://github.com/kubernetes-csi/external-snapshotter#usage](https://github.com/kubernetes-csi/external-snapshotter#usage)
 
-4. Configure storage system connectivity.
+5. (Optional) If planning on using volume replication (remote copy function), enable support on your Kubernetes cluster.
+
+    Install the following replication CRDs once per cluster.
+
+    ```
+    curl -O https://raw.githubusercontent.com/csi-addons/volume-replication-operator/v0.1.0/config/crd/bases/replication.storage.openshift.io_volumereplicationclasses.yaml
+    kubectl apply -f ./replication.storage.openshift.io_volumereplicationclasses.yaml
+    
+    curl -O https://raw.githubusercontent.com/csi-addons/volume-replication-operator/v0.1.0/config/crd/bases/replication.storage.openshift.io_volumereplications.yaml
+    kubectl apply -f ./replication.storage.openshift.io_volumereplications.yaml
+    ```
+
+6. (Optional) To use CSI Topology, any nodes in the cluster must have the following labels that introduce topology awareness:
+      - `topology.kubernetes.io/region`
+      - `topology.kubernetes.io/zone`
+
+      **Important:** These labels must be found on the nodes in the cluster **before** installing the IBM® block storage CSI driver. If these are not labeled before installation, the CSI driver will not be topology aware.
+
+      ```
+      $> kubectl get nodes -o=jsonpath='{range .items[*]}[{.metadata.name}, {.metadata.labels}]{"\n"}{end}' | grep --color "topology.block.csi.ibm.com"
+
+      [node1, {"beta.kubernetes.io/arch":"amd64","beta.kubernetes.io/os":"linux","kubernetes.io/arch":"amd64","kubernetes.io/hostname":"k8s-rhel-7-8-k8s-1-20-arbel-w1","kubernetes.io/os":"linux","topology.block.csi.ibm.com/demo-region":"demo-region-1","topology.block.csi.ibm.com/demo-zone":"demo-zone-1"}]
```

Comment:
@ArbelNathan do we really want your name in the official docs?

Reply:
oh, I guess not.. @rivkap can we change it?

Reply:
@ArbelNathan  of course. do you need something there or can I just delete that whole section?
`k8s-rhel-7-8-k8s-1-20-arbel-w1` can be either `k8s-rhel-7-8-k8s-1-20-w1` or `k8s-rhel-7-8-k8s-1-20-john-w1`. just let me know what is best

Reply:
I thought something with `demo` would be more consistent, but why do we even need this command?

Reply:
@oriyarde  per CSI-3241. @ArbelNathan  Please let me know what you want here.

Reply:
for node1 "kubernetes.io/hostname" will be "demo-node1", node2 will be "demo-node2" etc.

Reply:
DONE